README.md

```diff
@@ -93,45 +93,115 @@ Brainflow:
 To use parameters in within VRChat, write the parameter name as a path. For example, to get the left side alpha value, the parameter name would be:
 - `Brainflow/PowerBands/Left/Alpha`
 
-## Deprecation
+## Migration from Old Parameter Schema
 
-In order to use the old parameter names as documented in previous versions, add the argument `--use-old-reporter`. An announcement will be made soon to sunset the old parameter names.

```

Comment:
We should keep this section so this information stands out to users more, it's likely going to be pretty frustrating for them if they come here and can't immediately find out why their app isn't working properly